doc/admin-api.md

```diff
@@ -55,11 +55,12 @@
 |---------|---------|----|-----------|----|
 |name     |False |Auxiliary   |Identifies route names.|customer-xxxx|
 |desc     |False |Auxiliary   |route description, usage scenarios, and more.|customer xxxx|
-|uri      |True |Match Rules|In addition to full matching such as `/foo/bar`、`/foo/gloo`, using different [Router](architecture-design.md#router) allows more advanced matching, see [Router](architecture-design.md#router) for more.|"/hello"|
-|host     |False |Match Rules|Currently requesting a domain name, such as `foo.com`; pan-domain names such as `*.foo.com` are also supported.|"foo.com"|
-|hosts    |False |Match Rules|The `host` in the form of a list means that multiple different hosts are allowed, and match any one of them.|{"foo.com", "*.bar.com"}|
-|remote_addr|False |Match Rules|The client requests an IP address: `192.168.1.101`, `192.168.1.102`, and CIDR format support `192.168.1.0/24`. In particular, APISIX also fully supports IPv6 address matching: `::1`, `fe80::1`, `fe80::1/64`, etc.|"192.168.1.0/24"|
-|remote_addrs|False |Match Rules|The `remote_addr` in the form of a list indicates that multiple different IP addresses are allowed, and match any one of them.|{"127.0.0.1", "192.0.0.0/8", "::1"}|
+|uri      |True, can't be used with `uris` |Match Rules|In addition to full matching such as `/foo/bar`、`/foo/gloo`, using different [Router](architecture-design.md#router) allows more advanced matching, see [Router](architecture-design.md#router) for more.|"/hello"|
+|uris     |True, can't be used with `uri`|Match Rules|The `uri` in the form of an empty list means that multiple different hosts are allowed, and match any one of them.|["/hello", "/word"]|
```

Comment:
`different hosts are allowed` -> `different uris are allowed`

Reply:
> The `uri` in the form of an empty list

Should be non-empty list.

Reply:
Solved.